xray/commands/curation/audit.go

```diff
@@ -463,6 +471,14 @@ func (nc *treeAnalyzer) extractPoliciesFromMsg(respError *ErrorsResp) []Policy {
 			cond := polCond[1]
 			policies = append(policies, Policy{Policy: strings.TrimSpace(pol), Condition: strings.TrimSpace(cond)})
 		}
+		if len(polCond) == 4 {
+			pol := polCond[0]
+			cond := polCond[1]
+			exp := strings.ReplaceAll(strings.Replace(polCond[2], ": ", ":\n", 1), " | ", "\n")
+			rec := strings.ReplaceAll(strings.Replace(polCond[3], ": ", ":\n", 1), " | ", "\n")
```

Comment:
It is not clear what this replacement does. Let's export it to a private function to be more clear and remove the code duplication.